image-deployer-requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyyaml
+openstacksdk>1.0.1
```

Comment:
for right now, I'd add this to the main requirements.txt, as that's being referenced in the docker container.

Reply:
done